sharepoint-server-farm-ha/nic.json

```diff
@@ -0,0 +1,51 @@
+{
```

Comment:
I did this out of necessity rather than to improve modularity\reuse etc. In order to update a resource that I have previously created or updated in a template then I have to have a linked template to do this as it is not possible to specify the same resource twice in a template.

I can look at breaking this up into more subtemplates but I would rather do that as part of the refactoring once the issues with DSC extension are addressed